packages/cli/src/services/project.service.ts

```diff
@@ -71,6 +72,23 @@ export class ProjectService {
 		return project;
 	}
 
+	async updateProject(name: string, projectId: string): Promise<Project> {
+		const result = await this.projectRepository.update(
+			{
+				id: projectId,
+				type: 'team',
+			},
+			{
+				name,
+			},
+		);
+
+		if (!result.affected) {
+			throw new ApplicationError('Project not found', { level: 'warning' });
```

Comment:
Using `warning` level prevents this from ending up in Sentry

Reply:
Got it, what about returning 403 instead of 500?